Sources/TSPL/TSPL.docc/GuidedTour/GuidedTour.md

```diff
@@ -292,46 +296,45 @@ print(fruits)
   ```
 -->
 
-To create an empty array or dictionary,
-use the initializer syntax.
+To create an empty array, write `[]`
+and to create empty dictionary write `[:]` ---
```

Comment:
```suggestion
and to create empty dictionary, write `[:]` ---
```

I sorta feel like you need a comma here for consistency and maybe even a semicolon after the first "write []", but this is already a punctuation heavy sentence. Just a thought in case you wanted to think about reformulating this part.

Reply:
Yeah, that approach would want a comma there.  Tried a different, simpler, approach in 5dab09ba.